metrics/report/report_dockerfile/scaling.R

```diff
@@ -56,38 +57,100 @@ for (currentdir in resultdirs) {
 
 			testname=datasetname
 
-			cdata=data.frame(avail_gb=as.numeric(fdata$BootResults$mem_free$Result)/(1024*1024))
-			cdata=cbind(cdata, cpu_idle=as.numeric(fdata$BootResults$cpu_idle$Result))
+			cdata=data.frame(boot_time=as.numeric(fdata$BootResults$launch_time$Result)/1000)
+
+			# format the utilization data
+			udata=data.frame(nodename=fdata$BootResults$node_util)
+			for (i in seq(length(cdata[, "boot_time"]))) {
+				if (i == 1) {
+					# first iteration provide column name for c1
+					c1=cbind(node=udata$nodename.node)
+					c2=cbind(udata$nodename.cpu_idle$Result)
+					c3=cbind(udata$nodename.mem_free$Result)/(1024*1024)
+					c4=cbind(rep(i, length(udata$nodename.node)))
+					c5=cbind(rep(testname, length(udata$nodename.node)))
+					# FIXME figure out how to add schedule here ###
+					# then add to cdata below as well ###
+					# then update calculations to exclude schedule=false ###
+
+					# declare formatted utility data
+					fudata=cbind(c1,c2,c3,c4,c5)
+				}
+				else {
+					# shift to 0 based indexing
+					index=i-1
+					sindex=(index*3)+1
+					eindex=sindex+2
+					# grab 3 columns for next row bind
+					row=cbind(udata[,sindex:eindex])
+					c1=cbind(node=row[,1])
+					c2=cbind(row[,2]$Result)
+					c3=cbind(row[,3]$Result)/(1024*1024)
+					c4=cbind(rep(i, length(udata$nodename.node)))
+					c5=cbind(rep(testname, length(udata$nodename.node)))
+					# create the new row (which is actually the number of nodes of rows)
+					frow=cbind(c1,c2,c3,c4,c5)
+					fudata=rbind(fudata,frow)
+				}
+			}
+			colnames(fudata)=c("node", "cpu_idle", "mem_free", "pod", "testname")
+			# fudata is considered a vector for some reason so converting it to a data.frame
```

Comment:
I think that is because we initialise it with an empty vector using `=c()` higher up, whereas for other `data.frame` types, we initialise by first assigning them the result of a `data.frame()`, like:

```R
                        udata=data.frame(nodename=fdata$BootResults$node_util)
```
and then we append to those with `cbind()` etc.

Not a problem at all, just noting why it is a vector...